<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Digital Image Correlation Engine (DICe) 2D</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: #303030;
}
QMenuBar {
color: white;
background-color: #202020;
min-width: 50px;
}
QMenuBar::item {
spacing: 5px;
padding: 4px;
background: transparent;
border-radius: 4px;
}
QMenuBar::item:selected {
border: 1px solid yellow;
}
QMenuBar::item:pressed {
background: #888888;
}
QComboBox {
background-color: #404040;
color: white;
}
QComboBox QAbstractItemView{
background-color: #404040;
color: white;
}
QListView {
background-color: #404040;
border: none;
}
QTabWidget{
background-color: #202020;
}
QTabBar{
background-color: #202020;
}
QTabBar::tab{
background-color:#303030;
padding: 5px 15px 5px 15px;
color: white;
}
QTabBar::tab:hover:!pressed{
border: 1px solid yellow;
}
QTabBar::tab:selected{
background-color: #202020;
}
QTabWidget::pane{
border: none;
background-color: #202020;
}
QTextBrowser{
background-color: #303030;
color: white;
}
QTextEdit{
background-color: #303030;
color: white;
font: 12pt &quot;Courier&quot;;
}
QSpinBox {
color: white;
background-color: #404040;
}
QListWidget {
background-color: #404040;
color: white;
}
QCheckBox {
color: white;
}
QLineEdit {
background-color: #505050;
color: white;
}
QPushButton{
padding: 5px;
background-color: #808080; /*489ff9*/
border: 0px;
border-radius: 5px;
color: white;
height: 25px;
}
QPushButton:hover:!pressed
{
  border: 1px solid yellow;
}
QPushButton:pressed{
background-color: #505050;/*#3b83cd*/;
}
QPushButton:checked{
background-color: #3b83cd;
}
QPushButton#diceButton{
background-color: #101010;
}
QPushButton#diceButton:pressed{
background-color: #202020;
}
QPushButton#runButton{
font-weight: bold;
background-color: rgb(0, 255, 11);
color: black;
}
QPushButton#runButton:pressed{
background-color:rgb(39, 217, 24);
}
QPushButton#runButton:disabled{
background-color:#505050;
color: #808080;
}
QPushButton#boundaryPlus{
background-color:#1ec5c5;
}
QPushButton#boundaryMinus{
color:#efefef;
background-color:#168282;
}
QPushButton#excludedPlus{
background-color:#fc4bfc;
}
QPushButton#excludedMinus{
color:#efefef;
background-color:#9d349d;
}
QPushButton#boundaryPlus:checked{
border: 2px solid yellow;
}
QPushButton#excludedPlus:checked{
border: 2px solid yellow;
}
QPushButton#resetGeoButton{
background-color: rgb(173, 8, 49);
}
QPushButton#resetGeoButton:pressed{
background-color: rgb(163, 89, 99);
}
QPushButton#resetView{
background-color: rgb(21, 171, 255);
}
QPushButton#resetView:pressed{
background-color: rgb(23, 85, 122);
}
QLabel{
color: white;
background-color: #202020;
}
QGroupBox{
background-color: #202020;
color: white;
border: none;
padding: 10;
}
QProgressBar{
color: white;
}
QSelectionArea{
background-color: #404040;
}
QFrame#shapesFrame{
background-color: #404040;
}
QFrame#zoomFrame{
background-color: #404040;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>25</number>
    </property>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>ACTIONS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="inputTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="images">
       <attribute name="title">
        <string>Images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="workingDirButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Working Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="workingDirLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refFileButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Reference Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="defFileButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Deformed Images</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="defListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::SinglePass</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="defImageShow">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="defFileLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="parameters">
       <attribute name="title">
        <string>Parameters</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Optimization method</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QComboBox" name="optMethodCombo"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>75</height>
           </size>
          </property>
          <property name="title">
           <string>Interpolation method</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QComboBox" name="interpMethodCombo"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>75</height>
           </size>
          </property>
          <property name="title">
           <string>Initialization method</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QComboBox" name="initMethodCombo"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Shape functions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QCheckBox" name="translationShapeCheck">
             <property name="text">
              <string>Translation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="rotationShapeCheck">
             <property name="text">
              <string>Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="normalShapeCheck">
             <property name="text">
              <string>Normal Stretch</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="shearShapeCheck">
             <property name="text">
              <string>Shear Stretch</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Subset size</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QSpinBox" name="subsetSize"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Step size</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QSpinBox" name="stepSize"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="output">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>COORDINATES</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QCheckBox" name="xCheck">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="yCheck">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>y</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>MAPPING PARAMETERS</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QCheckBox" name="dispXCheck">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Displacement x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="dispYCheck">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Displacement y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="rotationCheck">
             <property name="text">
              <string>Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="normalXCheck">
             <property name="text">
              <string>Normal strectch x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="normalYCheck">
             <property name="text">
              <string>Normal stretch y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="shearCheck">
             <property name="text">
              <string>Shear stretch xy</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_11">
          <property name="title">
           <string>DIAGNOSTICS (smaller is better)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QCheckBox" name="sigmaCheck">
             <property name="text">
              <string>Sigma (displacement uncertainty)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="gammaCheck">
             <property name="text">
              <string>Gamma (match quality)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="betaCheck">
             <property name="text">
              <string>Beta (Objective sensitivity)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="statusCheck">
             <property name="text">
              <string>Status flag</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_12">
          <property name="title">
           <string>IMAGE QUALITY</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QCheckBox" name="numActiveCheck">
             <property name="text">
              <string>Num active pixels</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="contrastCheck">
             <property name="text">
              <string>Contrast level</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="noiseCheck">
             <property name="text">
              <string>Noise level</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_13">
          <property name="title">
           <string>STRAIN</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QCheckBox" name="strainCheck">
             <property name="text">
              <string>Compute strain</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="strainCombo"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Window size (pixels):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="strainWindowSpin"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="diceButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/resources/images/DICe_logo_white-01.png</normaloff>:/images/resources/images/DICe_logo_white-01.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>250</width>
        <height>75</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>PROGRESS</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QTextEdit" name="consoleEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="SimpleQtVTK" name="simpleQtVTKWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_input_files"/>
    <addaction name="actionLoad_working_dir"/>
    <addaction name="actionSet_backend_exec"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_DICe"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExport_input_files">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export input files...</string>
   </property>
  </action>
  <action name="actionAbout_DICe">
   <property name="text">
    <string>About DICe...</string>
   </property>
  </action>
  <action name="actionLoad_working_dir">
   <property name="text">
    <string>Import working dir...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionSet_backend_exec">
   <property name="text">
    <string>Set backend exec...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SimpleQtVTK</class>
   <extends>QWidget</extends>
   <header location="global">simpleqtvtk.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
